.github/workflows/robustness-template-arm64.yaml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
To make this work we are faced with the two choices:
  1. Update `robustness-template.yaml` to run all robustness jobs (not just `arm64`) in containers.
  2. Add a robustness template specifically for `arm64`.

As you will see in this pull request I've gone with option 2 as it is a more gradual change. We could revisit later on and adopt running jobs in containers more widely to consolidate once we know it's operating nicely.